module/bodh/request/validator/v1.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * name : v1.js
+ * author : Aman
+ * created-date : 06-04-2020
+ * Description : Bodh request validator.
+ */
+
+module.exports = (req) => {
+
+    let bodhRequestValidator = {
+        
+        middleware : function () {
+            req.checkBody('url').exists().withMessage("required url");
```

Comment:
@rakeshSgr - Check if the URL has a valid sunbird endpoint. Refer to Kendra for utils function